log/logwriter/writer_test.go

```diff
@@ -7,17 +7,20 @@ import (
 	"testing"
 	"time"
 
-	"github.com/bitrise-io/bitrise/log"
-	"github.com/bitrise-io/bitrise/log/logwriter"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+
```

Comment:
NIT: pls revert the import paths update, we only separate stdlib and other deps.